src/modules/roleMethods.ts

```diff
@@ -38,4 +40,20 @@ export const roleMethods = {
             take: data.take,
         });
     },
+
+    suggestions: async ({ query, include, take = suggestionsTake }: SuggestionsRole) => {
```

Comment:
getSuggestions

Reply:
same method in issues named suggestions, so I'd like to keep it name

Reply:
ok